test/formulator_test.exs

```diff
@@ -1,8 +1,37 @@
 defmodule FormulatorTest do
   use ExUnit.Case
   doctest Formulator
+  alias Phoenix.HTML.Form
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  describe "input - label" do
+    test "a label is created with the field name" do
+      [{:safe, label} | _] = %Form{data: %{name: ""}}
+      |> Formulator.input(:name)
```

Comment:
I'm not sure if you want to follow this style or not, but on some of our other projects we've been indenting the pipe if the output is assigned/pattern matched